src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,63 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  setOptionsValue: (event: string) => void;
+  setInputValue: (event: string) => void;
+  inputValue: string;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  setOptionsValue,
+  setInputValue,
+  inputValue,
+}) => {
+  function handleOptionValue(event: React.ChangeEvent<HTMLInputElement>) {
+    setOptionsValue(event.target.value);
```

Comment:
The event type for `handleOptionValue` should be `React.ChangeEvent<HTMLSelectElement>` instead of `React.ChangeEvent<HTMLInputElement>`, as it handles changes for a `<select>` element.